chatcommands.py

```diff
@@ -674,6 +674,16 @@ def master():
     os._exit(8)
 
 
+@command(privileged=True, aliases=['pip-update'])
```

Comment:
You don't need the `pip-update` alias - that's the name of the command, so it's already there.

```suggestion
@command(privileged=True)
```